pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/MailboxSendOperator.java

```diff
@@ -55,6 +54,10 @@
 public class MailboxSendOperator extends BaseOperator<TransferableBlock> {
   private static final Logger LOGGER = LoggerFactory.getLogger(MailboxSendOperator.class);
   private static final String EXPLAIN_NAME = "MAILBOX_SEND";
+  // TODO: Set this value via command line flag or query option.
+  // Max block size in bytes to send content over mail box service.
+  // Set to 4M for now.
+  private static int _maxBlockSize = 4 * 1024 * 1024;
```

Comment:
this might be a dynamic computed value depending on the mailbox sender method, e.g. if using HASH_DIST and assume evenly distributed, then it should be `single server mailbox block size * # of servers`
but we can simply add a TODO and fix later